packages/figma-design-tokens/input/tokens/$themes.json

```diff
@@ -2141,26 +2141,26 @@
-      "cmp.TextButton.Container.Padding_H.LG": "a02e98c9ed4fcac156c1ae3cf449685376d30549",
-      "cmp.TextButton.Container.Padding_H.XL": "bbcd50aea4a403b87b8ee0ca91a76ef5f7c76455",
-      "cmp.TextButton.Container.ItemSpacing.XS": "2fec083bb2468b793ccf370c8b95875548803952",
-      "cmp.TextButton.Container.ItemSpacing.SM": "15f56e2bb209b709c60befde44caa360183f507c",
-      "cmp.TextButton.Container.ItemSpacing.MD": "92e4aff96e9d82c9fc9160cb306473c85016bf81",
-      "cmp.TextButton.Container.ItemSpacing.LG": "657947f94af94e3c7bf51f285b1b5e620e5df367",
-      "cmp.TextButton.Container.ItemSpacing.XL": "02d34be610e173d9f38ee82a62dac9219b31593f",
-      "cmp.TextButton.Container.BorderRadius.XS": "6866a8f2c7c178bfdae89da7dea11c03198f7fda",
-      "cmp.TextButton.Container.BorderRadius.SM": "4accdfbfb18452648365b37bfe13ad1fb76c719e",
-      "cmp.TextButton.Container.BorderRadius.MD": "87f953f9321c05344ddc23aa744f4192bbd0b456",
-      "cmp.TextButton.Container.BorderRadius.LG": "effcfd0891bb13cf21b34990df479dd077c55e78",
-      "cmp.TextButton.Container.BorderRadius.XL": "aa1526d49944f9133ad50ec78e2c73df504b6489",
+      "cmp.ButtonText.Container.Padding_V.XS": "63334a1fcc8dfd4a5d72accef0044afbb2444745",
+      "cmp.ButtonText.Container.Padding_V.SM": "7507e04200aad7059cc9ff4116dcf20b633f9474",
+      "cmp.ButtonText.Container.Padding_V.MD": "022c1ee7a4a4788272bf3ec6a69f70bce6d826fd",
+      "cmp.ButtonText.Container.Padding_V.LG": "09ea8474d982ffd1b2f6171a154d049e97b88c45",
+      "cmp.ButtonText.Container.Padding_V.XL": "572ffa82a80abb0bd5d63bf40747f41757b4a122",
+      "cmp.ButtonText.Container.Padding_H.XS": "25f57dce3b86c32d80d86b0edd1856144976420e",
+      "cmp.ButtonText.Container.Padding_H.SM": "9c3d2dc1b77bd07b3863e35e49c153d3d1763d39",
+      "cmp.ButtonText.Container.Padding_H.MD": "f695cb375b789cba6c072072c2e69b23141f20fb",
+      "cmp.ButtonText.Container.Padding_H.LG": "a02e98c9ed4fcac156c1ae3cf449685376d30549",
+      "cmp.ButtonText.Container.Padding_H.XL": "bbcd50aea4a403b87b8ee0ca91a76ef5f7c76455",
+      "cmp.ButtonText.Container.ItemSpacing.XS": "2fec083bb2468b793ccf370c8b95875548803952",
+      "cmp.ButtonText.Container.ItemSpacing.SM": "15f56e2bb209b709c60befde44caa360183f507c",
+      "cmp.ButtonText.Container.ItemSpacing.MD": "92e4aff96e9d82c9fc9160cb306473c85016bf81",
+      "cmp.ButtonText.Container.ItemSpacing.LG": "657947f94af94e3c7bf51f285b1b5e620e5df367",
+      "cmp.ButtonText.Container.ItemSpacing.XL": "02d34be610e173d9f38ee82a62dac9219b31593f",
+      "cmp.ButtonText.Container.BorderRadius.XS": "6866a8f2c7c178bfdae89da7dea11c03198f7fda",
+      "cmp.ButtonText.Container.BorderRadius.SM": "4accdfbfb18452648365b37bfe13ad1fb76c719e",
+      "cmp.ButtonText.Container.BorderRadius.MD": "87f953f9321c05344ddc23aa744f4192bbd0b456",
+      "cmp.ButtonText.Container.BorderRadius.LG": "effcfd0891bb13cf21b34990df479dd077c55e78",
+      "cmp.ButtonText.Container.BorderRadius.XL": "aa1526d49944f9133ad50ec78e2c73df504b6489",
```

Comment:
@larserbach just to be sure here... renaming these as well is correct?

Reply:
Hey @ChristianHoffmannS2 , Hey @veilvokay ,

the tokens were not renamed on design side. Also we have not discussed if this should be done yet. Therefor I would slice this out of this PR. If we decide to also rename the tokens, then this should be done separately.

Reply:
@veilvokay , @ChristianHoffmannS2 , I discussed this topic with @larserbach today. We should not merge yet, but Lars will have a closer look and update the design-branch for the renaming. Once this is prepared, he will let us know to merge this topic.